File: src/main/java/de/tudresden/inf/lat/gentlerepair/plugin/GentleRepairTab.java
package de.tudresden.inf.lat.gentlerepair.plugin;

import org.protege.editor.owl.ui.OWLWorkspaceViewsTab;

public class GentleRepairTab extends OWLWorkspaceViewsTab {

  private static final long serialVersionUID = 2364494254783378893L;
//  private static final Logger log              = LoggerFactory.getLogger(GentleRepairTab.class);

  public GentleRepairTab() {
    setToolTipText("Custom tooltip text for Gentle Repair Tab");
  }

  @Override
  public void initialise() {
    super.initialise();
//    log.info("Gentle Repair Tab initialized");
  }

  @Override
  public void dispose() {
    super.dispose();
//    log.info("Gentle Repair Tab disposed");
  }

}

File: src/main/java/de/tudresden/inf/lat/gentlerepair/plugin/GentleRepairTab.scala
package de.tudresden.inf.lat.gentlerepair.plugin

import org.protege.editor.owl.ui.OWLWorkspaceViewsTab

@SerialVersionUID(2364494254783378893L)
class GentleRepairTab extends OWLWorkspaceViewsTab {

  this.setToolTipText("Custom tooltip text for Gentle Repair Tab")

  override def initialise(): Unit = {
    super.initialise()
    // log.info("Gentle Repair Tab initialized")
  }

  override def dispose(): Unit = {
    super.dispose()
    // log.info("Gentle Repair Tab disposed")
  }

}

